pkg/cluster/internal/create/actions/config/config.go

```diff
@@ -79,6 +79,7 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 			ControlPlaneEndpoint: controlPlaneEndpoint,
 			APIBindPort:          kubeadm.APIServerPort,
 			Token:                kubeadm.Token,
+			PodSubnet:            "10.244.0.0/16", // TODO: make configurable
```

Comment:
right, otherwise others CNI like Calico has to be configured to match that subnet in order to work

Reply:
Please, take a look at this approach to see if it makes sense https://github.com/kubernetes-sigs/kind/pull/348/commits/ef8d84bb33a314426e1c2705fad935191ac4b907
It allows the user to define the podSubnet but defaults to a const value

Reply:
going to punt making this configurable to a follow-up PR just because this is so big, but it's relatively trivial (mostly boilerplate) and I'll be able to file it immediately. this PR leaves all the hooks to inject a configured version

Reply:
https://github.com/kubernetes-sigs/kind/pull/524 is that follow-up